packages/central-server/src/kobo/startSyncWithKoBo.js

```diff
@@ -6,98 +6,128 @@
 import keyBy from 'lodash.keyby';
 import { DataBroker } from '@tupaia/data-broker';
 import { generateId } from '@tupaia/database';
+import winston from '../log';
 
 const PERIOD_BETWEEN_SYNCS = 10 * 60 * 1000; // 10 minutes between syncs
 const SERVICE_TYPE = 'kobo';
 
-export async function startSyncWithKoBo(models) {
```

Comment:
Don't be too scared! This file has mainly just had code moved around. The real changes are:
a) Switching to using the `data_service_sync_group` model. So now we sync for each DSSG separately from the others
b) Adding a bit more logging to the `sync_group_log` table

Reply:
I love a PR with a don't be scared disclaimer 😆